src/main/java/com/zeiterfassung/hwr/desktop/component/LoginPane.java

```diff
@@ -44,33 +50,67 @@ public Parent getParent()
         passwordField.setPromptText("Password");
         btnSubmit = new Button("login");
         btnSubmit.getStyleClass().add("redButton");
+        errorLabel = new Label("Fehler");
+        errorLabel.setVisible(false);
+        errorLabel.getStyleClass().add("errorLabel");
         vBox = new VBox();
-        vBox.getChildren().addAll(textFieldEmail, passwordField, btnSubmit);
+        vBox.getChildren().addAll(textFieldEmail, passwordField, btnSubmit, errorLabel);
         borderPane = new BorderPane();
         borderPane.setLeft(new ImageView(image));
         borderPane.setCenter(vBox);
 
         btnSubmit.setOnAction(click ->
         {
-            if (isUserValid())
-            {
-                Button btn = (Button) click.getSource();
-                Scene scene = btn.getScene();
-                Stage stage = (Stage) scene.getWindow();
-                Scene nextScene = new Scene(nextPane.getParent());
-                nextScene.getStylesheets().add("static/styling.css");
-                stage.setScene(nextScene);
-            }
+            login.setEmail(textFieldEmail.getText());
+            login.setPassword(hash(passwordField.getText()));
+
+            WebClient.create("http://localhost:8080/login")
```

Comment:
host sollte in config File ausgelagert werden

Reply:
jo refactoring steht noch aus inkl. MVC implementieren